.github/workflows/active_stash_smoke_test.yml

```diff
@@ -0,0 +1,54 @@
+name: "ActiveStash: Smoke Test"
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+jobs:
+  active-stash-smoke-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    env:
+      BUNDLE_GEMFILE: active_stash_smoke_test.gemfile
+      RSPEC_FULL_BACKTRACE: yes
+      CS_IDP_CLIENT_SECRET: ${{ secrets.CS_IDP_CLIENT_SECRET }}
+      CS_WORKSPACE: ${{ secrets.CS_WORKSPACE }}
+      CS_KMS_KEY_ARN: ${{ secrets.CS_KMS_KEY_ARN }}
+      CS_NAMING_KEY:  ${{ secrets.CS_NAMING_KEY }}
+      RAILS_VERSION: 6
```

Comment:
Do we need to test against Rails 7 as well?

Reply:
We could test against multiple Rails versions here, but I don't think it's super helpful for the ActiveStash smoke tests in the ruby-client repo. ruby-client itself doesn't depend on Rails, so I think it's alright if we leave testing multiple Rails versions to ActiveStash CI.